src/api/payment-requests/payment-requests.md

```diff
@@ -109,15 +109,15 @@ Once you have made that payment, you can use the transaction id to [Pay a Paymen
 
 ### Payment Condition
 
-Some asset types require conditional approval to pay. Possible payment conditions include confirming proof of ID
+Some [Asset Types][] require conditional approval to pay. Possible Payment Conditions include confirming proof of ID
 or confirming a promotional item was purchased.
 
-The `conditionsEnabled` flag should be set to true when [Creating a Payment Request](#create-a-payment-request) or these asset
-types may not be available as a [Payment Option](#payment-option).
+The `conditionsEnabled` flag should be set to true when [Creating a Payment Request](#create-a-payment-request) to indicate that
+Payment Conditions can be accepted. If a Payment Condition arises, the absence of the `conditionsEnabled` flag will result in the
+Payment Request being cancelled.
 
-Conditions can either be [accepted](#accept-payment-condition) or [declined](#accept-payment-condition).
-Conditions must be accepted for a Payment Request to be `paid` with an asset type that requires conditional approval.
```

Comment:
I think this is implied by the first sentence "Some asset types require conditional approval to pay" therefore I removed it to make this section easier to read.